src/qutip_qip/decompositions/decompositions_extras.py

```diff
@@ -0,0 +1,26 @@
+from qutip_qip.decompositions.general_decompositions import (check_input,
+check_input_shape, convert_qobj_gate_to_array)
+
+from qutip_qip.decompositions.single_decompositions import (_ZYZ_rotation, ABC_decomposition)
+from qutip import *
```

Comment:
In general, one should avoid star importation in a package. `qutip` is a very large namespace. It includes a lot of things in the namespace that you may not be aware of.

Personally, I find it ok to do this at the early stage of development while exploring different things. But should be removed before merging.